DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/DYSAC/DYSAC.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>DYSAC</id>
-    <version>1.0.0.13</version>
+    <version>1.0.0.16</version>
```

Comment:
Please correct version number format
Format is 'Major Verson'.0. 'SprintNumber'. 'Iteration'
1.0.16.1

Reply:
Can I assume that this should be DYSAC sprint number?

Reply:
yes, that is correct

Reply:
I have created a new package `1.0.11.1` since this is the sprint DYSAC is in.